cms/static/js/spec/views/settings/main_spec.js

```diff
@@ -56,8 +58,34 @@ define([
             // It sends `POST` request, because the model doesn't have `id`. In
             // this case, it is considered to be new according to Backbone documentation.
             AjaxHelpers.expectJsonRequest(
-                requests, 'POST', '/course-details', expectedJson
+                requests, 'POST', urlRoot, expectedJson
             );
         });
+
+        it('Creating an entrance exam with minimum_score_pct. It should save the ' +
```

Comment:
Note that the test name is meant to read as a sentence starting with the 'it'. I see that the pre-existing test in this file doesn't follow this convention either.